core/src/main/java/org/jboss/pnc/causeway/pncclient/PncBuild.java

```diff
@@ -2,9 +2,34 @@
 
 import org.jboss.pnc.causeway.rest.BrewNVR;
 
+import java.util.ArrayList;
+import java.util.List;
+
 public final class PncBuild {
 
+    public final List<PncArtifact> buildArtifacts;
+    public final List<PncArtifact> dependencies;
+
+    public PncBuild() {
+        buildArtifacts = new ArrayList<>();
+        dependencies = new ArrayList<>();
+    }
+
     public BrewNVR createNVR() {
         return null;
     }
+
+    public static class PncArtifact {
```

Comment:
Why not just use the ArtifactRest stuff directly here? It seems like extra overhead to maintain an extra set of model classes for a REST endpoint that's pretty PNC-specific, isn't it? Also, I think we're going to need the origin URLs so we can pass the artifact binaries through.